dear_petition/petition/api/viewsets.py

```diff
@@ -245,6 +247,62 @@ def create(self, request):
         return resp
 
 
+def download_csv(modeladmin, request, queryset):
```

Comment:
Is this non-class `download_csv` method used anywhere?

Reply:
Ah, no. I stopped and started a little bit while working on this. Good catch.